src/time_integration/methods_2N.jl

```diff
@@ -42,9 +48,14 @@ struct CarpenterKennedy2N54 <: SimpleAlgorithm2N
 end
 
 """
-      CarpenterKennedy2N43()
+    CarpenterKennedy2N43()
+The following structures and methods provide a minimal implementation of
```

Comment:
```suggestion

The following structures and methods provide a minimal implementation of
```
I think we need a newline here.